types/wlr_output.c

```diff
@@ -231,14 +231,15 @@ void wlr_output_destroy(struct wlr_output *output) {
 
 void wlr_output_effective_resolution(struct wlr_output *output,
 		int *width, int *height) {
```

Comment:
Maybe it's worth it to have `double`s here too?

Reply:
Hmm, I'm not sure we get any benefit from that. This isn't a place where loss of precision impacts anything.

Reply:
Well, we use it below for the cursor code:

https://github.com/swaywm/wlroots/pull/329/files#diff-8c8783fe608815c9626588d85b00f2f4R273

What we basically do is: `(int)(width / scale) * scale`.

Reply:
Yeah, but there are no displays (or if there are, I've never heard of them) with odd dimensions so you are always going to be dealing with integers here anyway.

Reply:
Okay, so we should be fine for now. We won't be anymore when we'll introduce fractional scaling.